### ISSUE TITLE
App is crashing on uploading video

### ISSUE THREAD

**Issue by Preetinegi11** [clickntap/Vimeo#25]
It is showing this error.any one knows how to resolve this issue:-
No static field INSTANCE of type Lorg/apache/http/message/BasicLineFormatter;

**Comment by saurabhrai-o** [comment]
@Preetinegi11 Yes i have also begin to see this problem from last friday i.e. May 1st

Any luck resolving this?

**Comment by saurabhrai-o** [comment]
> 
> 
> It is showing this error.any one knows how to resolve this issue:-
> No static field INSTANCE of type Lorg/apache/http/message/BasicLineFormatter;

@Preetinegi11 Yes i have also begin to see this problem from last friday i.e. May 1st
Any luck resolving this?

**Comment by Hassan-Elseoudy** [comment]
@Preetinegi11 can you show us your code? 
it's working fine to me.
